catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/dialog/NewProjectDialogFragment.java

```diff
@@ -122,10 +122,15 @@ void showOrientationDialog(String projectName, boolean createEmptyProject) {
 
 	void createDroneProject(String name, boolean jumpingSumo) {
 		try {
-			ProjectManager.getInstance().initializeNewProject(name, getActivity(), false, true,
-					false, false, jumpingSumo);
+			if (jumpingSumo) {
```

Comment:
i personally like the version of this function as it was before better, maybe just renaming of the variable JumpingSumo to something like isJumpingSumoProject.
But not necessary

Reply:
if (not_necessary) {
please 
please 
please 
don't make effort to write a useless comment. 
thanks x^n
}
done (renamed to isJumpingSumoProject).